demo_path_routing_auth/tools.py

```diff
@@ -1,3 +1,7 @@
+# The CSV also specifies which tool goes with each file... but I could imagine
+# listing multiple incompatible input sets there. The default_files here can
+# just be a simple, coherent example to start from.
```

Comment:
I think I'll just delete the comment. I had been thinking about defining the default inputs for each tool in this csv as well, but it would make the code more complicated, and avoid just a tiny bit of redundancy. The comment was mostly me trying to justify this choice to myself, and isn't really that useful. Thanks!